lib/logger.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import logging.config
+
+class SeleniumFilter(logging.Filter):
+    def filter(self, record):
+        selenium_log = 'selenium.webdriver.remote.remote_connection'
+        return selenium_log not in record.getMessage()
+
+# much of this taken from http://victorlin.me/posts/2012/08/26/good-logging-practice-in-python
+# reference -- https://docs.python.org/2/library/logging.config.html
+logging.config.dictConfig({
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "simple": {
+            "format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        },
+        "console": {
+            "format": "%(message)s"
+        }
+    },
+
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "INFO",
+            "formatter": "console",
+            "stream": "ext://sys.stdout"
+        },
+
+        "debug_file_handler": {
+            "class": "logging.handlers.RotatingFileHandler",
+            "level": "DEBUG",
+            "formatter": "simple",
+            "filename": "debug.log",
+            "maxBytes": 400000,
+            "backupCount": 20,
+            "encoding": "utf8"
+        },
+
+        "info_file_handler": {
+            "class": "logging.handlers.RotatingFileHandler",
+            "level": "INFO",
+            "formatter": "simple",
+            "filename": "info.log",
+            "maxBytes": 100000,
+            "backupCount": 20,
+            "encoding": "utf8"
+        },
+
+        "error_file_handler": {
+            "class": "logging.handlers.RotatingFileHandler",
+            "level": "ERROR",
+            "formatter": "simple",
+            "filename": "errors.log",
+            "maxBytes": 50000,
+            "backupCount": 20,
+            "encoding": "utf8"
+        }
+    },
+
+    "loggers": {
+        "__main__": {
+            "level": "INFO",
+            "handlers": ["console"]
+        }
+    },
+
+    "root": {
+        "level": "DEBUG",
+        "handlers": ["debug_file_handler", "info_file_handler", "error_file_handler"]
+    }
```

Comment:
These are the loggers. The top part is for custom loggers, which I made for `__main__` (which is what `pucauto.py` gets called when you run `python pucauto.py`).

The bottom part is the `root` logger, which every single logger will use. This makes it so that anything that gets logged in `pucauto.py` uses the console handler, which doesn't print debug statements and only prints messages. This also means that it'll use the root handler too. So every time you call `LOGGER.info('something')` in `pucauto.py`, you get it printed out simply in the console, and a more verbose version of that in `info.log`.